wfe/wfe.go

```diff
@@ -1753,12 +1799,14 @@ func (wfe *WebFrontEndImpl) validateAuthzForChallenge(authz *core.Authorization)
 	defer authz.RUnlock()
 
 	ident := authz.Identifier
-	if ident.Type != acme.IdentifierDNS {
+	switch ident.Type {
```

Comment:
Given that there is no body for most of the cases in this switch I think I'd prefer to see it written as an `if` statement like:

`if ident.Type != acme.IdentifierDNS && ident.Type != acme.IdentifierIP {`